src/components/Visualizations/Graph/Graph.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
```

Comment:
I'm actually going to extend this component further as more functionality is required so I don't have to deal with the library as much.